### PR TITLE
Added experimental data for 3-methyl-heptane

### PR DESCRIPTION
**Pull request by ndharms**
Manually obtained data from DOI: 10.1016/j.proci.2012.06.001 for ignition delays from RPI and NUIG shocktube experiments.

### CODE REVIEW

**Comment by kyleniemeyer** [comment]
Hi @nateharms, thanks for the submission! A few comments:

1. I noticed that, in the first file at least, in the `authors` block you just have the first author and then "et al.". For the files to validate properly, each author needs to be in that list, like 
```
  authors:
    - name: Gaurav Mittal
    - name: Chih-Jen Sung
      ORCID: 0000-0003-2046-8076
    - name: Richard A Yetter
```
2. which leads to my second question: did you run [pyked](https://github.com/pr-omethe-us/PyKED) to validate the files?
3. Did the associated paper give any information about uncertainty for any quantities?

**Comment by ndharms** [comment]
@kyleniemeyer 

Sorry for the delay in responding. 

1. I fixed the "et al." issue
2. I was able to validate these file using pyked after fixing the issue
3. There were no uncertainties given in the supplemental information

Also, I organized these data in the proper `file-authors` format as opposed to the old `file-author` format.

**Comment by bryanwweber** [comment]
@nateharms The uncertainties are given on page 337 of the PDF article.

**Comment by ndharms** [comment]
@bryanwweber @kyleniemeyer 

Okay, third time's the charm. 🤞 Let me know if there are any additional changes I need to make!

**Comment by bryanwweber** [comment]
Looks good now. Any reason not to include the RCM data from the same study?

**Comment by ndharms** [comment]
I was just unsure about how to write those data in the proper format. If y'all could point me to an example of an RCM file, I can get those written and either make another second pull request for those data or include them in this request (depending on what y'all think is the right way to go about including those data).

**Comment by bryanwweber** [comment]
I think they'll look exactly the same as the shock tube files, with the exception of swapping out the experiment type and ignition definition. As best I can tell, the volume/pressure traces haven't been provided anywhere, so can't include them...

File: 3-methyl-heptane/Wang_phi_0.5_m-h_0.0122_xO2_0.2075_NUIG.yaml [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        - 1311.0 kelvin
+      ignition-delay:
+        - 54.0 us
+        - uncertainty-type: relative
+          uncertainty: 0.2
+      pressure:
+        - 8.24 atm
+      composition: *comp
+      ignition-type: *ign
+      equivalence-ratio: 0.5
```
Comment (bryanwweber) [inline 3-methyl-heptane/Wang_phi_0.5_m-h_0.0122_xO2_0.2075_NUIG.yaml]: Can you make sure to have a newline at the end of each file?

Comment (rwest) [reply 3-methyl-heptane/Wang_phi_0.5_m-h_0.0122_xO2_0.2075_NUIG.yaml]: I know you said a new line, not two new lines, but it still reminds me of this wonderful RMG feature: https://github.com/ReactionMechanismGenerator/RMG-Py/issues/798
But RMG is in good company. Gaussian also requires blank lines at the end of the input file; the program will crash and dump core if they are missing. 😭

Comment (bryanwweber) [reply 3-methyl-heptane/Wang_phi_0.5_m-h_0.0122_xO2_0.2075_NUIG.yaml]: https://xkcd.com/1172/

Comment (bryanwweber) [reply 3-methyl-heptane/Wang_phi_0.5_m-h_0.0122_xO2_0.2075_NUIG.yaml]: But ending the file with a newline is a git best practice, nothing to do with our format (AFAIK!) 😃 See: https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file and especially the comments on the first answer

**Comment by kyleniemeyer** [comment]
@nateharms should this have been closed? we never merged it

**Comment by ndharms** [comment]
Oh, sorry, I was changing around a lot of stuff today and I think I accidentally closed it. 

I haven't added RCM data yet though.

**Comment by kyleniemeyer** [comment]
@nateharms ok cool, just let us know when it's ready for another review

**Comment by ndharms** [comment]
@kyleniemeyer @bryanwweber

Sorry about the delay for getting this commit up, but I added the RCM data and verified it. I did have a question about something. In the supplemental information, there was a reference to first stage heat loss... how would I add that to corresponding data points?

**Comment by kyleniemeyer** [comment]
@nateharms do you mean that they provide a volume history for the RCM files? The best practice for RCM is to include a volume (or pressure) history that accounts for heat loss. If they give it in some other way, it may need to be converted.

Also, I noticed that a few of the files still don't have a newline at the end (or maybe those are new files), could you add that? (FYI, some text editors like Atom will do that automatically for you 😄.)

**Comment by ndharms** [comment]
@kyleniemeyer 

Attached is the supporting information for RCM data and below is an excerpt from the paper on the RCM calculations... Thoughts on how to convert it?

screen shot 2018-08-27 at 2 59 07 pm


> RCM measurements carried out at NUIG utilized a fixed compression ratio of 10.2 and a compression time of 16.6 ms. For the fixed compression ratio the initial temperature was varied from 373 to 413 K, providing compressed test gases at 10 atm in two temperature windows from 825 to 899 K for the pure N2 diluent and 714–782 K for a 50/50 N2/CO2 diluent mixture. Compressed gas conditions were calculated using ideal theory based on the initial conditions and compression ratio. Ignition delay times, based on measured pressure profiles, were defined as the time interval from the end of compression to the maximum rate of pressure rise at ignition. Uncertainties in RCM ignition delay times are estimated at ±15%, based on contributions from uncertainties in compressed gas conditions and pressure measurements.

**Comment by kyleniemeyer** [comment]
Heat loss in units of time... ? I admit I don't know what's going on there.

@bryanwweber do you have any idea what that means?

I do appreciate that they estimated uncertainties in ignition delay at 15%, though I wonder how they got that number.

**Comment by bryanwweber** [comment]
@kyleniemeyer @nateharms I have never seen that before. Better email the authors to figure out what's going on.

**Comment by ndharms** [comment]
@kyleniemeyer @bryanwweber Gotcha, I emailed the corresponding author. I'll let y'all know what I hear back. Also, I made sure all of my files have the single line at the end of them.

**Comment by ndharms** [comment]
From the author, although.. I'm still not sure how to convert the RCM data....

> Nathan,
> 
> The RCM experiments that you are trying to model were performed at NUIG by Henry Curran’s group. I’ve copied Henry here. Perhaps he can share an appropriate pressure or volume history with you.
> 
>  
> 
> For RCM experiments there is heat loss to the walls that can be modeled as an adiabatic volume expansion. See this paper by Henry Curran’s group https://www.sciencedirect.com/science/article/pii/S0010218008000394. For your simulations you will need a measured pressure profile for the NUIG RCM for non-reacting conditions that can then be converted to a volume profile. There is a pressure history in the above linked article you may be able to convert.
> 
>  
> 
> Best Regards,
> Matt

**Comment by bryanwweber** [comment]
Hi @nateharms! That response is how RCMs are typically modeled. Including the pressure trace would be sufficient for our purposes.

Unfortunately, it looks like a lot of data has been mixed up in this pull request. Can you rebase/squash out any commits that don't add 3-methyl-heptane data? If you need help with that, let us know!

**Pull request closed without merging**